dhis-2/dhis-api/src/test/java/org/hisp/dhis/scheduling/JobConfigurationSerializationTest.java

```diff
@@ -52,6 +52,10 @@
  */
 public class JobConfigurationSerializationTest
 {
+    private static final String PROGRAM1 = "Information Campaign";
```

Comment:
It does not affect the test, but would be nice to use UIDs instead of names to make the test closer to the real implementation, which uses UID now.

Reply:
Agree with @maikelarabori on this one.

Reply:
Done